devTools/phpstan-src.neon

```diff
@@ -62,14 +61,14 @@ parameters:
             message: "#Do not use magic number (.*)#"
             count: 8
             path: ../src/Mutator/Extensions/BCMath.php
+        -
+            identifier: arrayFilter.strict
```

Comment:
same question: is it because it produces too many errors or is it because we don't want this rule at all?

Reply:
It produces 8 new errors for cases like this https://github.com/infection/infection/blob/dfe9cf6e65545881c7d21343c494cc8a1fdbfb80/src/Configuration/Schema/InvalidSchema.php#L58

Do we want to define callback explicitly?

Reply:
I would vote no ;)